src/nvim/popupmnu.c

```diff
@@ -910,10 +910,18 @@ void pum_set_event_info(dict_T *dict)
   if (!pum_visible()) {
     return;
   }
-  tv_dict_add_nr(dict, S_LEN("height"), pum_height);
-  tv_dict_add_nr(dict, S_LEN("width"), pum_width);
-  tv_dict_add_nr(dict, S_LEN("row"), pum_row);
-  tv_dict_add_nr(dict, S_LEN("col"), pum_col);
+  int w,h,r,c;
+  if (!ui_pum_get_pos(&w, &h, &r, &c)){
```

Comment:
`ui_pum_get_pos` should probably return the pos for builtin pum as well. Then we don't need to duplicate the `dict_add` code.

Reply:
🆗, `ui_pum_get_pos` logic updated.